server/__tests__/view-group-messages-tests.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-undef */
+const request = require('supertest');
+const { Message } = require('../models');
+
+const app = require('../server');
+const { HttpError } = require('../utils/errors');
+
+let server;
+jest.mock('../models', () => ({
+  Message: jest.fn()
+}));
+jest.mock('../utils/token-helpers', () => ({
+  authenticateToken: jest.fn().mockImplementation((req, res, next) => {
+    const user = {
+      id: '1'
+    };
+    req.user = user;
+    next();
+  }),
+}));
+beforeAll(() => {
+  server = app.listen(3001);
+});
+
+afterAll(() => {
+  server.close();
+});
+
+const token = 'token';
+
+const baseURL = 'http://localhost:3001';
+
+describe('When a user views messages in a group', () => {
+  it('should return correct status code if the user is viewing messages', async () => {
+    Message.mockImplementationOnce(() => ({
+      view: jest.fn(() => Promise.resolve()),
+    }));
+    const response = await request(baseURL).get('/api/group/:id/messages').set('Authorization', `Bearer ${token}`);
+    expect(response.statusCode).toBe(200);
```

Comment:
Here's some pseudo code for the your reference
```suggestion
    expect(response.statusCode).toBe(200);
     expect(response.data.messages).toBe([message]);
```